src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,22 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = (number, props) => {
```

Comment:
Our components should only ever take one argument, props. See my comment below about how we should have done the map.